.autover/changes/d0822afd-daf4-437a-9437-2a9492c135b7.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Projects": [
+    {
+      "Name": "Amazon.Lambda.RuntimeSupport",
```

Comment:
Did you mean to have 2 entries for RuntimeSupport? Seems like both can be combined into 1.

Reply:
Fixed. The second one is supposed to be `Amazon.Lambda.Core` which I marked as `preview` since the API is in preview status.